app/views/components/_figure.html.erb

```diff
@@ -6,7 +6,6 @@
   <img class="app-c-figure__image" src="<%= src %>" alt="<%= alt %>">
   <div class="app-c-figure__figcaption-wrapper">
     <figcaption class="app-c-figure__figcaption">
```

Comment:
Given the distinct lack of captions for our image formats, is it worth conditionally adding the `<div class="app-c-figure__figcaption-wrapper">` element?